docs/quickstart.md

```diff
@@ -0,0 +1,614 @@
+content-type: application/json
+content-length: 2
+access-control-allow-origin: *
+access-control-allow-headers: Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, Api-Version, Response-Time
+access-control-allow-methods: POST, GET, HEAD
+access-control-expose-headers: Api-Version, Request-Id, Response-Time
+content-md5: 11FxOYiYfpMxmANj4kGJzg==
+date: Mon, 28 Jul 2025 14:38:39 GMT
+server: cloudapi/9.20.0
+api-version: 9.0.0
+request-id: 6bf0b965-d49b-40c0-8619-675c02afa305
+response-time: 320
+
+[]
+# as expected your account those not have access keys, so let's generate them.
+cloudapi /your-manta-account/accesskeys -X POST
+HTTP/1.1 201 Created
+location: /neirac/accesskeys/91f41250ffff1bb1472611dd3b0bde50
+content-type: application/json
+content-length: 346
+access-control-allow-origin: *
+access-control-allow-headers: Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, Api-Version, Response-Time
+access-control-allow-methods: POST, GET, HEAD
+access-control-expose-headers: Api-Version, Request-Id, Response-Time
+content-md5: IWO6EvVxPH542Zv+Q9fAAA==
+date: Mon, 28 Jul 2025 14:38:51 GMT
+server: cloudapi/9.20.0
+api-version: 9.0.0
+request-id: 9fd136ad-1021-4d55-a257-aeeffdac2071
+response-time: 373
+
+{"dn":"accesskeyid=your-access-key-id, uuid=your-manta-account-uuid, ou=users, o=smartdc","controls":[],"accesskeyid":"your-access-key-id","accesskeysecret":"your-access-key-secret","created":"2025-07-28T14:38:51.900Z","objectclass":"accesskey","status":"Active"}
+
+```
+
+### Step 2: Configure Your S3 Client
+
+#### AWS CLI Configuration
+
+```ini
+[default]
+aws_access_key_id = "your-manta-access-key"
+aws_secret_access_key = "your-manta-secret-key"
+region = us-east-1
+s3=
+         addressing_style = path
+```
+
+#### s3cmd Configuration
+
+Create or edit `~/.s3cfg`:
+
+```ini
+[default]
+access_key = your-access-key
+secret_key = your-secret-key
+host_base = your-manta-endpoint
+host_bucket = your-manta-endpoint
+use_https = True
+signature_v2 = False
+
+```
+
+#### Environment Variables
+
+You can also use environment variables:
+
+```bash
+export AWS_ACCESS_KEY_ID=your-access-key
+export AWS_SECRET_ACCESS_KEY=your-secret-access-key
+export AWS_DEFAULT_REGION=us-east-1
+export AWS_ENDPOINT_URL=https://your-manta-endpoint
+```
+
+## Testing Your Configuration
+
+### Test Basic Operations
+
+```bash
+# List buckets
+aws s3 ls --endpoint-url https://your-manta-endpoint
+
+# Create a bucket
+aws s3 mb s3://test-bucket --endpoint-url https://your-manta-endpoint
+
+# Upload a file
+aws s3 cp local-file.txt s3://test-bucket/ --endpoint-url https://your-manta-endpoint
+
+# List objects in bucket
+aws s3 ls s3://test-bucket/ --endpoint-url https://your-manta-endpoint
+
+# Download a file
+aws s3 cp s3://test-bucket/local-file.txt downloaded-file.txt --endpoint-url https://your-manta-endpoint
+```
+
+### Test with ACLs
+ACLs are not applied to objects within a bucket, not to the bucket itself.
```

Comment:
ACLs are not applies to objects NOR the bucket?  This feels wrong compared to the code I'd been reviewing. The sentence above is very confusing.